src/debug/netcoredbg/vscodeprotocol.cpp

```diff
@@ -279,6 +276,14 @@ typedef std::function<HRESULT(
     const json &arguments,
     json &body)> CommandCallback;
 
+void VSCodeProtocol::addCapabilities(json &capabilities)
```

Comment:
How about renaming it to `AddCapabilitiesTo` ?
* Capital `A` follows the naming convention for other class methods
* `To` indicates that the this methods adds capabilities to passed argument instead of adding capabilities to class instance